app/models/existing_role.rb

```diff
@@ -0,0 +1,34 @@
+class ExistingRole < ActiveRecord::Base
```

Comment:
Role model already exist , and it keeps which role is assigned to which user.(user-role mapping).